packages/model-viewer/src/test/helpers.ts

```diff
@@ -225,4 +225,58 @@ export const createFakeThreeGLTF = () => {
     } as unknown as GLTFParser,
     userData: {}
   };
-};
\ No newline at end of file
+};
+
+export interface scenarioConfig {
+  name: string;
+  model: string;
+  lighting: string;
+  dimensions: {width: number, height: number};
+  target: {x: number, y: number, z: number};
+  orbit: {theta: number, phi: number, radius: number};
+  verticalFoV: number;
+  renderSkybox: boolean;
+}
+;
+
+export interface FidelityResult {
+  colorlessPixelCount: number;
+  transparentPixelCount: number;
+}
+;
+
+export const SCENARIOS: Array<scenarioConfig> = [
+  {
+    name: 'khronos-MetalRoughSpheres',
+    model:
+        'models/glTF-Sample-Models/2.0/MetalRoughSpheres/glTF/MetalRoughSpheres.gltf',
+    lighting: 'environments/lightroom_14b.hdr',
+    dimensions: {width: 100, height: 100},
```

Comment:
the origin dimension is 768, 768, im just using an arbitrary small dimension here to make the test faster, so feel free to change it.